src/utils/helpers.ts

```diff
@@ -0,0 +1,8 @@
+import { Tab } from '../types/Tab';
+
+export const findCorrectTab = (
+  initTabs: Tab[],
+  selectedId: string,
+): Tab | undefined => {
```

Comment:
```suggestion
): Tab => {
```

You are always returning a Tab